package.json

```diff
@@ -40,14 +40,13 @@
     "*.js": "npm run lint:fix"
   },
   "dependencies": {
-    "@skyscanner/eslint-config-skyscanner": "^15.0.0",
-    "@skyscanner/stylelint-plugin-backpack": "^2.0.1",
-    "stylelint": "^14.16.1",
-    "stylelint-config-prettier": "^9.0.4",
```

Comment:
Stylelint v15 deprecates rules that conflict with Prettier, so `stylelint-config-prettier` is no longer needed